cloud_functions/token_refresh/main.py

```
@@ -0,0 +1,46 @@
+            db.collection("users")
+            .document(uid)
+            .get()
+            .to_dict()[f"stream={data_source}"]
+        )
+
+        if not refresh_token:
+            raise ValueError(f"No refresh token found for user {uid}")
+        if data_source == "strava":
+            strava_refresh_oauth_token(db, uid, stream_data["refreshToken"])
+            return {"status": "success", "uid": uid}
+        else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)